privacyidea/lib/resolvers/SQLIdResolver.py

```diff
@@ -126,17 +126,19 @@ def _append_where_filter(conditions, table, where):
         :return: list of filter conditions
         """
         if where:
-            # this might result in errors if the
-            # administrator enters nonsense
-            (w_column, w_cond, w_value) = where.split()
-            if w_cond.lower() == "like":
-                conditions.append(getattr(table, w_column).like(w_value))
-            elif w_cond == "==":
-                conditions.append(getattr(table, w_column) == w_value)
-            elif w_cond == ">":
-                conditions.append(getattr(table, w_column) > w_value)
-            elif w_cond == "<":
-                conditions.append(getattr(table, w_column) < w_value)
+            parts = where.split(" and ")
```

Comment:
Many people use a capital "AND" in an SQL syntax. Do you thing we should split for ``" AND "``. Currently I can not think about an _easy_ way to split for lower or upper AND.

Reply:
We could also use ``re.split``:
```python
>>> x = 'a == x and b LIKE c AND c == d aNd foo < bar'
>>> re.split(' and ', x, flags=re.IGNORECASE)
['a == x', 'b LIKE c', 'c == d', 'foo < bar']
```